spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/AbstractClientConnectionFactory.java

```diff
@@ -126,9 +139,14 @@ protected TcpConnectionSupport buildNewConnection() throws Exception {
 	 * @param socket The new socket.
 	 */
 	protected void initializeConnection(TcpConnectionSupport connection, Socket socket) {
```

Comment:
I don't remember, but it's available for subclasses to customize, but of course, they would be required to call `super`.

We can think about it.